release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -24,8 +24,9 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines.
 //Removed the note per https://issues.redhat.com/browse/GRPA-3517
 
-//TODO: Add this for 4.14
-Starting with {product-title} 4.12, an additional six months is added to the Extended Update Support (EUS) phase on even numbered releases from 18 months to two years. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+Starting with {product-title} 4.12, Red Hat has added a six months Extended Update Support (EUS) phase for even numbered releases, extending the total available lifecycle from 18 months to two (2) years. In the {product-title} 4.12 release, this additional EUS is only available for the `x86_64` architecture.
```

Comment:
```suggestion
Starting with {product-title} 4.12, Red Hat has added a 6-month Extended Update Support (EUS) phase for even numbered releases, extending the total available lifecycle from 18 months to 2 years. In the {product-title} 4.12 release, this additional EUS is only available for the `x86_64` architecture.
```
Based on the[ IBM style guide](https://www.ibm.com/docs/en/ibm-style?topic=measurement-numerals-versus-words), I believe you would want to be consistent and use numerals throughout the entire sentence.